### DISCUSSION TITLE
Trying to add mcstatus capability to turing-smart-screen-python...

### DISCUSSION THREAD

**Discussion by wanderling** [mathoudebine/turing-smart-screen-python#310] [General]
Full disclosure, I am trying to do this while learning python as well. I don't do anything in half measures, lol.

I have most of the code lined out and 'appearently' error free, but the turing scripts are not using the data I am force feeding it. I would like someone with experience in either turing directly or just with python to assess my try and give me useful critiques. I can either post each of the files code here or upload it to github as my own variant of turing-smart-screen-python. Whichever would be more useful. I just don't want to make my broken code public until I make it work in the basic capacity if possible.

My drive for doing this is to 'ironically' encourage my little boy to learn how to code...

Thanks in advance.

Daniel

**Comment by ghost** [comment]
You'll have to be willing to disclose at least part of the source code if you want anyone else to review it. Ideally it would be all of it because you usually need the whole program to do any debugging.

**Reply by wanderling** [reply]
I am absolutely willing to disclose all of the modified code. My question was where would it be most helpful for someone to diagnose. Here as attachments or inline code, or as a fork of the main branch?

What I didn't want to do is release broken code to the world until it worked (at least partially). But if a fork is best, so be it.

**Reply by ghost** [reply]
Try a temporary branch under your fork repository. And I honestly don't think you need to worry about "releasing broken code" as it is in a place where people are unlikely to find. There's always going to be mess involved in development, or as others would call them, "rough drafts". You just need to reach a point where you have something refined enough to put into a PR.

**Comment by wanderling** [comment]
https://github.com/wanderling/turing-smart-screen-python/tree/turing-smart-screen-python-with-mcstatus. Appreciate the look. I hope it's something simple.
